### ISSUE TITLE
SNS 2?

### ISSUE THREAD

**Issue by philipyoungg** [tamtamchik/apple-iap-tools#5]
Thank you for this! I assume this is based on SNS 1 right?

Any plan for 2nd version?

**Comment by tamtamchik** [comment]
@philipyoungg yep, this is for V1. Frankly speaking, I didn't know that they had moved to V2. Thank you! 

I will take a closer look at changes in V2 next week and probably bump the package version as well.

**Comment by tamtamchik** [comment]
Just to keep everyone updated: it's WIP at the moment. Changes in Apple's API are very complex, so I'm working on covering with types all those 3 APIs related to Server Notifications and Receipt Validation:
- https://developer.apple.com/documentation/appstoreserverapi
- https://developer.apple.com/documentation/appstorereceipts
- https://developer.apple.com/documentation/appstoreservernotifications

Also will probably add JWS support and helpers to work with JWS based objects.
